app/views/main/dashboard.html.erb

```diff
@@ -15,9 +15,38 @@
           <td>
             <div class="btn-group" role="group" aria-label="Validate or Remove Teacher">
               <%= button_to("Validate", validate_path(:id => teacher.id),
-                 class: 'btn btn-success', type: 'button') %>
-              <%= button_to("Deny", deny_path(:id => teacher.id),
-                 class: 'btn btn-danger', type: 'button') %>
+                class: 'btn btn-success', type: 'button') %>
+              <span>
+                <button class="btn btn-danger" type="button" data-toggle="modal" data-target="<%= "#denialModal" + teacher.id.to_s %>" >
+                  Deny
+                </button>
+              </span>
+              <div class="modal fade" id="<%= "denialModal" + teacher.id.to_s %>" tabindex="-1" role="dialog" aria-labelledby="denialModalLabel" aria-hidden="true">
+                <div class="modal-dialog">
+                  <div class="modal-content">
+                    <div class="modal-header">
+                      <h4 class="modal-title">Denial</h4>
```

Comment:
Minor thing, but if you have the chance, could we add a the teacher name to the modal? Using the previous `data-` mechanism should work.